Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -233,7 +233,7 @@ - (void)downloadObjectServer {
                        @"--no-save",
                        @"--no-package-lock",
                        @"install",
-                       [@"realm-object-server@" stringByAppendingString:desiredVersion]
+                       [@"mongodb-stitch" stringByAppendingString:@"@3.11.0"]
```

Comment:
This should be `desiredVersion` (which is read from dependencies.list), not a hardcoded version.